test/spec/core/CanvasSpec.js

```diff
@@ -1098,6 +1110,26 @@ describe('Canvas', function() {
 
     }));
 
+
+    it('switches to correct layer', inject(function(canvas) {
+
+      // given
+      canvas.setActivePlane('a');
+      var shape = canvas.addShape({
```

Comment:
Interesting design decision manifests here: Rather than explicitly stating onto which plane to draw we do that implicitly.

Explicit drawing on a specific layer is possible and the recommended way to do things (cf. bpmn-js). I presume we handle that by specifying a root element, correct?

      var shape = canvas.addShape({
        id: 's0',
        x: 0, y: 0,
        width: 10, height: 10
      }, canvas.getPlane('a').rootElement);
```

Reply:
Correct, that's the way [I plan to do it in bpmn-js](https://github.com/bpmn-io/bpmn-js/commit/24b1885aa6d029170d076f71601374712a2315cb#diff-d1f7a9e332bbbcbb5d7f4365bca90e54c84d0a86106a89ece7708e453ef77b67). I'll adjust the tests to reflect that